Analysis/HiggsTauTau/src/HTTRun2AnalysisTools.cc

```diff
@@ -64,16 +64,46 @@ namespace ic {
+      alias_map_["vbf_run1"] = "(n_jets>=2 && jdeta>3.5 && mjj>500 && n_bjets==0 && n_jetsingap==0)";
+      alias_map_["vbf_tight_run1"] = "(n_jets>=2 && jdeta>4. && mjj>700 && n_bjets==0 && n_jetsingap==0 && pt_tt>100)";
+      alias_map_["vbf_loose_run1"] = "(!"+alias_map_["vbf_tight_run1"]+" && n_jets>=2 && jdeta>3.5 && mjj>500 && n_bjets==0 && n_jetsingap==0)";    
+      if(ch_ == channel::et){
+        alias_map_["1jet_loose_run1_et"] = "(!"+alias_map_["vbf_run1"]+" && n_jets>=1 && n_bjets==0 && met>30 && pt_2<=45))";
+        alias_map_["1jet_tight_run1_et"] = "(!"+alias_map_["vbf_run1"]+" && n_jets>=1 && n_bjets==0 && met>30 && pt_tt>100 && pt_2>45))";
+      } else if(ch_ == channel::mt){
+        alias_map_["1jet_loose_run1"] = "(!"+alias_map_["vbf_run1"]+" && n_jets>=1 && n_bjets==0 && pt_2<=45)";
+        alias_map_["1jet_medium_run1"] = "(!"+alias_map_["vbf_run1"]+" && n_jets>=1 && n_bjets==0 && pt_2>45 && pt_tt<=100)";
+        alias_map_["1jet_tight_run1"] = "(!"+alias_map_["vbf_run1"]+" && n_jets>=1 && n_bjets==0 && pt_tt>100 && pt_2>45)";
+      }
+      alias_map_["0jet_loose_run1"] = "(n_jets==0 && pt_2<=45)";
+      alias_map_["0jet_tight_run1"] = "(n_jets==0 && pt_2>45)";
+    }
+    else if (ch_ == channel::tt){
+      alias_map_["vbf_run1_tt"] = "(n_jets>=2 && jdeta>3.5 && mjj>500 && n_bjets==0 && pt_tt>100 && n_jetsingap==0)";
+      alias_map_["1jet_run1_tt"] = "(!"+alias_map_["vbf_tt_run1"]+" && n_jets>=1 && n_bjets==0 && pt_tt>100)";
+      alias_map_["1jet_loose_run1_tt"] = "(!"+alias_map_["vbf_tt_run1"]+" && n_jets>=1 && n_bjets==0 && pt_tt>100 && pt_tt<=170)";
+      alias_map_["1jet_tight_run1_tt"] = "(!"+alias_map_["vbf_tt_run1"]+" && n_jets>=1 && n_bjets==0 && pt_tt>170 )";
+      alias_map_["0jet_run1_tt"] = "(!"+alias_map_["vbf_tt_run1"]+" && !"+alias_map_["1jet_run1_tt"]+" && n_jets>=0 && n_bjets==0)";
+    }
+
 
     if (ch_ == channel::et || ch_ == channel::mt) {
       // SM Categories
+      //SM Categories
+        
       alias_map_["inclusive"]         = "1";
       alias_map_["nojet"]           ="(n_jets==0)";
       alias_map_["njet"]           ="(n_jets>0)";
       alias_map_["notwoprong"]       ="(tau_decay_mode_2!=6&&tau_decay_mode_2!=5)";
-      alias_map_["baseline"]         = "(iso_1<0.1 && mva_olddm_tight_2>0.5 && antiele_2 && antimu_2 && !leptonveto)";
+      alias_map_["baseline"]         = "(iso_1<0.1 && mva_olddm_medium_2>0.5 && antiele_2 && antimu_2 && !leptonveto)";
+      
       if(ch_ == channel::mt && year_.find("6")!=year_.npos) alias_map_["baseline"] = "(iso_1<0.15 && mva_olddm_medium_2>0.5 && antiele_2 && antimu_2 && !leptonveto)";
-      if(ch_ == channel::et && year_.find("6")!=year_.npos) alias_map_["baseline"] = "(iso_1<0.1  && mva_olddm_medium_2>0.5 && antiele_2 && antimu_2 && !leptonveto)";
+      if(ch_ == channel::et && year_.find("6")!=year_.npos) alias_map_["baseline"] = "(iso_1<0.1  && mva_olddm_tight_2>0.5 && antiele_2 && antimu_2 && !leptonveto)";
```

Comment:
So for the sm we don't know for sure what the baseline WP will be. I set this to tight as my studies of different WP suggested this was most optimal for et. Maybe we need option for sm/mssm since you use medium for mssm?